docs/components/input.md

```diff
@@ -272,18 +336,56 @@ Fired when the text starts or stops exceeding the limit, returns an object:
 </KComponent>
 ```
 
+### `icon:click`
```

Comment:
Two issues:

1. Can we make the emit `click:icon` instead? This way when it's bound to the component, it will look like `@click:icon` which seems more standard.
2. I don't see where you actually added the `emit` in the component.

Reply:
1. Respectfully disagree - we have similar pattern in `KTable` and we use `@row:click` and `@cell:click` there.
2. The component doesn't emit the event - it calls bound callback function directly (again, similar pattern we have in `KTable`) - basically a workaround I had to do because adding 'icon:click' to emits will remove it from attributes so isIconClickable.value always returns false. Let me know if this is confusing